libtock/crt0.c

```diff
@@ -326,9 +328,7 @@ void _c_start_pic(uint32_t app_start, uint32_t mem_start) {
   }
 
   main();
```

Comment:
What is `return 1` is used in `main()`?

I'm fully onboard with removing this `while (1)` loop and having apps do that if they want to, but this should be `exit(main())`